samples/drivers/watchdog/src/iwdg_main.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2017 RnDity Sp. z o.o.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <watchdog.h>
+#include <misc/printk.h>
+
+/*
+ * The independent watchdog (IWDG) is clocked by its own dedicated
+ * 40kHz low-speed internal clock (LSI).
+ *
+ * Prescaler register should be set to 'divider /16' and the reload
```

Comment:
Mention this example is for F3 family, or provide calculation steps depending on LSI clock